test/test_mps.py

```diff
@@ -10276,11 +10270,76 @@ def get_samples():
                 self.assertEqual(cpu_out, mps_out, atol=atol, rtol=rtol)
 
             except Exception as e:
-                if any(s in str(e).lower() for s in ["int64", "float16", "div truc rounding"]):
-                    self.skipTest(f"Expected Runtime Error: {str(e)}")
+                raise e
```

Comment:
Remove the try/except altogether? If you want to keep it here to make the diff easier to read and remove it in a follow up that doesn't change any logic, I'm happy with that as well ;)

Reply:
Done.